apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -95,54 +76,78 @@ $(function() {
                         <div class="{{block_formatting}} pl-order-block" style="margin-left: {{indent}}px;flex-grow:1;">{{{inner_html}}}</div>
                     </div>
                 </li>
-                {{/distractor_feedback}}
-            {{/student_submission}}
-        </ul>
-    </div>
-    <p>{{{feedback}}}</p>
-    {{/parse-error}}
+            </ul>
+            <div>
+                <span class="pl-order-blocks-feedback-icon"><i class="fa fa-question-circle" aria-hidden="true"></i></span>
+                {{distractor_feedback}}
+            </div>
+        </li>
+        {{/distractor_feedback}}
+        {{^distractor_feedback}}
+        <li class="{{#is_disordered}}{{#disorder_feedback}}pl-order-blocks-pairing-indicator list-group-item-warning{{/disorder_feedback}}{{/is_disordered}}">
+                <div class="pl-order-blocks-correctness-badge-container">
+                    {{#allow_feedback_badges}}
+                    <span class="pl-order-blocks-correctness-badge badge {{badge_type}}"><i class="fa {{icon}}" aria-hidden="true"></i></span>
+                    {{/allow_feedback_badges}}
+                    <div class="{{block_formatting}} pl-order-block" style="margin-left: {{indent}}px;flex-grow:1;">{{{inner_html}}}</div>
+                </div>
+            {{#is_disordered}}
+                {{#disorder_feedback}}
+                <div>
+                    <span class="pl-order-blocks-feedback-icon"><i class="fa fa-question-circle" aria-hidden="true"></i></span>
+                    This block is out of order. Hint: {{disorder_feedback}}
```

Comment:
I'm unsure about prefixing the feedback with this fixed text. Is there a reason we wouldn't just display the instructor-provided feedback verbatim?